vmprof/reader.py

```diff
@@ -43,12 +71,12 @@ class JittedCode(int):
     pass
 
 def wrap_kind(kind, pc):
-    if kind == VMPROF_ASSEMBLER_TAG:
-        return AssemblerCode(pc)
-    elif kind == VMPROF_JITTED_TAG:
-        return JittedCode(pc)
-    assert kind == VMPROF_CODE_TAG
-    return pc
+    cls = {
+        VMPROF_ASSEMBLER_TAG: AssemblerCode,
+        VMPROF_JITTED_TAG: JittedCode,
+        VMPROF_CODE_TAG: lambda x: x,
+    }[kind]
```

Comment:
Yes. The performance increase of writing it down with explicit if/elses is negligible. I did a quick profiling and `wrap_kind` doesn't even make 0.1% of the overall parse time.